### DISCUSSION TITLE
VCS-Integration stopped working

### DISCUSSION THREAD

**Discussion by CarolusMagnuss** [joyfullservice/msaccess-vcs-addin#303] [Q&A] [answered]
Hello,

I have sucessfully used this Add In for some months now and I am very satisfied with the results. However a recent, faulty Microsoft Patch broke things for me and i haven't been able to get things running again. I am talking about this issue: https://www.devhut.net/another-access-bug-cannot-open-any-more-databases/

I added the Database location to the trusted locations as suggested, which let them delete the locking files again. The Add-In is unfortunatetly not working, giving me an error, that the "assistant" is not installed or corrupted. The addin is not showing in the Add In Manager anymore and trying to add it manually gives me an error saying that an error occurred and the add in hasn't been added to the Microsoft Access Directory (see picture below). This doesn't give me anything to go by. Do you have any suggestions to fix this problem? 

Regards

![image](https://user-images.githubusercontent.com/40948921/152700287-a7bfbf1b-9ac9-4bf0-b563-d8a8e1665f75.png)

**Comment by joyfullservice** [comment]
Hi @CarolusMagnuss, 

Sorry to hear of the issues you are having there! I wanted to let you know that we have an issue in this repository where we are tracking this specific problem. Please feel free to post any additional details there.

- #302 

I also wanted to point out that the access-vcs-integration add-in will not show up in the the Add-in Manager. I used to install it that way several years ago, but we have made some major improvements to the installer so it can be installed by non-admin users and works much better than using the Add-in Manager.

To resolve the problem you are having with the add-in, I would try downloading it from GitHub and installing it again. That should correct any problems with file corruption or missing trusted locations.

I hope that helps!

**Reply by CarolusMagnuss** [reply]
Thanks a lot for your suggestion. I did already reinstall the plug in, but it didn't not help. Fortunately the newest build fixed the bug.

**Comment by hecon5** [comment, marked as answer]
It appears this bug has been fixed by Microsoft in Build 14827.20192; hopefully this gets you back on your way!

**Reply by CarolusMagnuss** [reply]
Thanks for the Info, it did indeed and i am relieved i dont have to go back to making regular copies of the Development folder. A Pull of the most recent build and compiling of that version also fixed some form of remnant bug in the database.
Thanks again for your time.
